src/models/User.js

```diff
@@ -0,0 +1,46 @@
+class User {
+    _id;
+    _pesel;
+    _firstName;
+    _lastName;
+    _dateOfBirth;
+    _mail;
+    _password;
```

Comment:
Czy na pewno chcemy kiedykolwiek przechowywać na frntendzie zahashowane hasło usera? nie jest to szczególnie bezpieczne, po tej stronie wyciął bym to pole

Reply:
tak jest w dokumentacji, ale tymczasowo można je usunąć, jeżeli pojawi się potrzeba, to można to przywrócić